.github/ISSUE_TEMPLATE/feature---enhacement-proposal.md

```diff
@@ -16,7 +16,7 @@ Proposals not following the template below will be closed immediately.
 
 **Describe the problem or limitation you are having in your project:**
 
-**Describe how this feature / enhancement will help you overcome this problem or limitation:**
+**Describe the feature / enhancement and how will it help to overcome the problem or limitation:**
```

Comment:
```suggestion
**Describe the feature / enhancement and how it helps to overcome the problem or limitation:**
```
Depending on your mindset. 🙂

Reply:
The suggested change is much better. The original isn't proper English.

Reply:
Learning something new each time, updated.